models_files/preprocessing.py

```diff
@@ -0,0 +1,31 @@
+#import all necessary packages
+import nltk
+nltk.download('punkt')
+from nltk.stem import PorterStemmer
+from nltk.tokenize import word_tokenize
+from nltk.corpus import stopwords
+nltk.download('stopwords')
```

Comment:
Maybe push the downloaded softwares beforehand?

Reply:
okay